pkg/query-service/app/cloudintegrations/controller.go

```diff
@@ -117,7 +118,7 @@ func (c *Controller) GenerateConnectionUrl(
 	}
 
 	// TODO(Raj): parameterized this in follow up changes
-	agentVersion := "0.0.3"
+	agentVersion := "v0.0.4-alpha.14"
```

Comment:
make change to support this through config.

Reply:
It should be, but not right now, there's no updation strategy in-place.

Roadmap acc to me would be = Adding more AWS Integrations -> Azure Integrations -> Filters in AWS integrations (If possible in azure too) -> Updation Strategy for AWS and Azure

Reply:
I meant that we should take it from the signoz deployment config instead of hardcoded value.